telecine/src/main/java/com/jakewharton/telecine/TelecineApplication.java

```diff
@@ -31,10 +32,28 @@
       Timber.plant(tree);
     }
 
-    objectGraph = ObjectGraph.create(new TelecineModule(this));
+    telecineModule = new TelecineModule(this);
+
+    initTelecineComponent();
+
+    getTelecineComponent().inject(this);
```

Comment:
Delete this and the corresponding method on the component. This type doesn't require injection.